src/libraries/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -34,6 +34,22 @@ public static IEnumerable<object[]> Names_TestData()
             yield return new object[] { "\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B", "\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B" };
         }
 
+        public static IEnumerable<object[]> Names_TestDataQuoted()
```

Comment:
Why quoted -- would something like `Names_TestDataRequiresEscaping()` be better?

Reply:
Updated the name